2024/08/13 13:05:52 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.23.0.1, server: _, request: "GET / HTTP/1.1", upstream: "fastcgi://172.23.0.4:9000", host: "localhost"
2024/08/13 13:05:54 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.23.0.1, server: _, request: "GET / HTTP/1.1", upstream: "fastcgi://172.23.0.4:9000", host: "localhost"
2024/08/13 13:05:56 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.23.0.1, server: _, request: "GET / HTTP/1.1", upstream: "fastcgi://172.23.0.4:9000", host: "localhost"
2024/08/13 13:11:38 [error] 28#28: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.25.0.1, server: _, request: "GET / HTTP/1.1", upstream: "fastcgi://172.25.0.4:9000", host: "localhost"
2024/08/13 13:11:40 [error] 28#28: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.25.0.1, server: _, request: "GET / HTTP/1.1", upstream: "fastcgi://172.25.0.4:9000", host: "localhost"
2024/08/13 13:41:44 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: _, request: "GET / HTTP/1.1", upstream: "fastcgi://172.27.0.4:9000", host: "localhost"
2024/08/13 13:41:45 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: _, request: "GET / HTTP/1.1", upstream: "fastcgi://172.27.0.4:9000", host: "localhost"
2024/08/13 13:41:47 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: _, request: "GET / HTTP/1.1", upstream: "fastcgi://172.27.0.4:9000", host: "localhost"
2024/08/13 14:07:52 [error] 28#28: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.0.1, server: _, request: "GET / HTTP/1.1", upstream: "fastcgi://172.30.0.4:9000", host: "localhost"
2024/08/13 14:07:53 [error] 28#28: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.0.1, server: _, request: "GET / HTTP/1.1", upstream: "fastcgi://172.30.0.4:9000", host: "localhost"
2024/08/13 14:07:54 [error] 28#28: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.0.1, server: _, request: "GET / HTTP/1.1", upstream: "fastcgi://172.30.0.4:9000", host: "localhost"
2024/08/13 14:07:55 [error] 28#28: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.0.1, server: _, request: "GET / HTTP/1.1", upstream: "fastcgi://172.30.0.4:9000", host: "localhost"
2024/08/13 14:07:56 [error] 28#28: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.0.1, server: _, request: "GET / HTTP/1.1", upstream: "fastcgi://172.30.0.4:9000", host: "localhost"
